testdata/simple/main.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 Google Inc. All rights reserved.
```

Comment:
ah ok I see. this seems a bit out of the realm of a "structure test" to me, but you could probably accomplish it with the structure test framework. don't think there's anything stopping you from just issuing the compile command through a command test (probably as a setup command), and then another command that actually calls the binary and checks the output to ensure it worked properly. I have plans to add support for checking the error code of a command test, which seems like an easy way for you guys to make sure your program worked correctly.